mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/master/resourcecluster/ClusterID.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.mantisrx.server.master.resourcecluster;
+
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+/**
+ * Represents the cluster that the task executor belongs to.
+ */
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@Value
+public class ClusterID {
+  String resourceID;
```

Comment:
this should be private?

Reply:
value classes by default in Lombok treat the fields as "private final". 

Here's the documentation: https://projectlombok.org/features/Value